src/test/resources/json/html.json

```diff
@@ -0,0 +1,71 @@
+[
```

Comment:
The current sample json doesn't contain enough html.

I'd also rather not modify it as FeaturesOverviewPageIntegrationTest#generatePage_generatesStatsTableFooter and some other tests rely on statistics of that exact file.